src/configs/index.js

```diff
@@ -0,0 +1,54 @@
+import ImportContactsIcon from '@material-ui/icons/ImportContacts';
+
+const languages = ['uk', 'en'];
+
+const routes = {
+  pathToNews: '/',
+  pathToNewsDetails: '/news/:id',
+  pathToAddNews: '/newsadd'
+};
+
+const theme = {
+  DARK_THEME: 'dark',
+  LIGHT_THEME: 'light'
+};
+
+const buttonTitles = {
+  DELETE_TITLE: 'Видалити',
+  EDIT_TITLE: 'Редагувати',
+  CREATE_NEWS_TITLE: 'Додати новину'
+};
+
+const statuses = {
```

Comment:
Please, remove these variables and move its values to the object `config` where it is used

Reply:
Removed!